doc/nrf/releases_and_maturity/software_maturity.rst

```diff
@@ -1881,6 +1881,59 @@ PSA Crypto support
          - Supported
          - Supported
 
+
+MCUboot bootloader
```

Comment:
The MCUboot bootloader shouldn't be a section under Security Feature Support. This should be a separate new section.